spec/01-unit/08-router_spec.lua

```diff
@@ -3571,6 +3570,66 @@ for _, flavor in ipairs({ "traditional", "traditional_compatible", "expressions"
           end
         end
       end)
+
+      it("works with special characters('\"','\\')", function()
+        local use_case_routes = {
+          {
+            service    = {
+              name     = "service-invalid",
+              host     = "example.org",
+              protocol = "http"
+            },
+            route      = {
+              id = "e8fb37f1-102d-461e-9c51-6608a6bb8101",
+              paths    = { [[/\d]] },
+            },
+          },
+          {
+            service    = {
+              name     = "service-invalid",
+              host     = "example.org",
+              protocol = "https"
+            },
+            route      = {
+              id = "e8fb37f1-102d-461e-9c51-6608a6bb8102",
+              paths    = { [[~/\d+"]] },
+            },
+          },
+        }
+
+        local router = assert(Router.new(use_case_routes))
```

Comment:
should use function `new_router` to test flavor `expressions`.

Reply:
I'm just moving this part of code.